check.sh

```diff
@@ -67,6 +67,16 @@ function findGodot() {
         echo "Found 'godot4' executable"
         godotBin="godot4"
 
+    elif command -v godot &>/dev/null; then
+        # Check if `godot` actually is Godot 4.x
+        if godot --version | grep -qE "^4\\."; then
+            echo "Found 'godot' executable with version $(godot --version)"
+            godotBin="godot"
+        else
+            echo "Godot executable found, but it is the incompatible version $(godot --version)"
+            exit 2
+        fi
```

Comment:
This should not fail while there are other options to exhaust (the `.bat` one).
I think the whole `godot` check should come last.

Reply:
Good point. Will change that.

Reply:
Or, the `elif command -v godot &>/dev/null` and `if godot --version | grep -qE "^4\\."` should be combined to a single condition.

Reply:
My hunch would be the separate clauses improve usability here as they allow us to print the diagnostic in the case the user tries to use gdext with Godot 3 installed. Otherwise they would just get "Godot executable not found" and be none the wiser

Reply:
Makes sense.